Content.Client/Lobby/UI/LobbyGui.xaml

```diff
@@ -14,7 +14,7 @@
                  Stretch="KeepAspectCovered" />
     <BoxContainer Name="MainContainer" VerticalExpand="True" HorizontalExpand="True" Orientation="Horizontal"
                   Margin="10 10 10 10" SeparationOverride="2">
-        <SplitContainer State="Auto" HorizontalExpand="True">
```

Comment:
Can't you use `Resize="False"`?

Reply:
Oh, I didn't know I could comment on the deleted side, weird

Reply:
🤷

Reply:
okay...